tools/changelog-generator/app/generate-full-changelog.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+source ${APP_PATH}/variables.sh
+
+CONFIGURE_GIT=$1
+if [ "$CONFIGURE_GIT" = "--configure-git" ]; then
+    echo "Configuring git..."
+    source ${APP_PATH}/config-git.sh
+fi
```

Comment:
Splitting code to this line to separate file and use it in `tools/changelog-generator/app/generate-full-changelog.sh` and `tools/changelog-generator/app/generate-release-changelog.sh` would be great 👍